homeassistant/components/media_player/kodi.py

```diff
@@ -15,20 +19,49 @@
     SUPPORT_TURN_OFF, MediaPlayerDevice, PLATFORM_SCHEMA)
 from homeassistant.const import (
     STATE_IDLE, STATE_OFF, STATE_PAUSED, STATE_PLAYING, CONF_HOST, CONF_NAME,
-    CONF_PORT, CONF_USERNAME, CONF_PASSWORD)
+    CONF_PORT, CONF_USERNAME, CONF_PASSWORD, EVENT_HOMEASSISTANT_STOP)
 import homeassistant.helpers.config_validation as cv
 
-REQUIREMENTS = ['jsonrpc-requests==0.3']
+REQUIREMENTS = ['websockets==3.2']
```

Comment:
aiohttp appears to be hanging [here](https://github.com/KeepSafe/aiohttp/blob/v1.2.0/aiohttp/client.py#L302), if I just execute `http http://192.168.0.23:9090/jsonrpc` in a shell I see `HTTP/1.1 400 Bad Request` then it eventually times out as the socket complete it's response (probably poor behaviour from Kodi).

I'm probably seeing `400` because I'm not sending the correct headers, and I'm wondering what headers [websockets](https://github.com/aaugustin/websockets/blob/master/websockets/client.py#L46) is sending that aiohttp isn't for the handshake request. 

Just trying to determine what we could do to resolve the issue.